canonical-websites/docs.jujucharms.com
# Update templates to look like docs.jujucharms.com

Update base template and error pages with basically the same markup as what's on docs.jujucharms.com now.

I added extra CSS to style the nav, which is in a different markup format.

I also removed all Sass building - as in this version the CSS is included in the markup, as per docs.jujucharms.com. I'm not planning to update this to build the CSS properly from vanilla with Sass, because the whole styling will be rewritten before long to go into jaas.ai/docs.

Fixes https://github.com/ubuntudesign/base-squad/issues/431

QA
--

`./run`, load the site. See it looks a lot like https://docs.jujucharms.com (apart from the homepage).
